### ISSUE TITLE
Can I use hover intent for better interaction with dropdown menus?

### ISSUE THREAD

**Issue by jakemauer** [movableink/ember-pop-over#37]
Currently I have some menus appearing on hover using `[hover, hold]` but due to the space between the activator button and the dropdown menu, if you're not quick the menu will disappear before your mouse pointer gets there. 

Is there a way to utilize the existing hover intent service for this purpose?

**Comment by tim-evans** [comment]
Yes! You can make the hover intent smarter by detecting wether you're moving the cursor towards the pop over.

**Comment by tim-evans** [comment]
(PRs welcome)
